src/ui/settings/ViewSettingsMenu.qml

```diff
@@ -47,10 +47,18 @@ VPNFlickable {
             }
 
             VPNUserProfile {
-                _iconButtonImageSource: "qrc:/nebula/resources/chevron.svg"
-                _iconButtonOnClicked: () => {
-                    Sample.manageAccountClicked.record();
-                    VPN.openLink(VPN.LinkAccount)
+                property bool subscriptionManagementEnabled: VPNFeatureList.get("subscriptionManagement").isSupported
+                _iconButtonImageSource: subscriptionManagementEnabled
+                    ? "qrc:/nebula/resources/chevron.svg"
+                    // TODO: Show a loading indicator for VPNProfileFlow.StateLoading
+                    : "qrc:/nebula/resources/open-in-new.svg"
+                _iconButtonOnClicked: () => {                    
+                    if (subscriptionManagementEnabled) {
+                        VPNProfileFlow.start();
```

Comment:
should we call `reset` here?

Reply:
Calling `reset` now when starting the `VPNProfileFlow`.